include/globals.h

```diff
@@ -1736,9 +1775,8 @@ inline bool SetSocketBlockingEnabled(int fd, bool blocking)
 
 inline String formatSize(size_t size, size_t threshold = 1000)
 {
-    // If the size is less than the threshold, we don't need to worry about precision because
-    // we'll be showing whole units
-    const int precision = size >= threshold ? 2 : 0;
+    // If the size is above the threshold, we want a precision of 2 to show more accurate value
+    const int precision = size < threshold ? 0 : 2;
```

Comment:
Check me here, I could be crazy, but I think this was backwards, so flipping it

Reply:
I think I'm neutral on the matter.

I am bound to remember this exchange if you change your mind about this in a couple of years, though. ;)